docker/README.md

```diff
@@ -214,38 +227,48 @@ To create mar [model archive] file for TorchServe deployment, you can use follow
 1. Start container by sharing your local model-store/any directory containing custom/example mar contents as well as model-store directory (if not there, create it)
 
 ```bash
-docker run --rm -it -p 8080:8080 -p 8081:8081 --name mar -v $(pwd)/model-store:/home/model-server/model-store -v $(pwd)/examples:/home/model-server/examples  pytorch/torchserve:latest
+docker run --rm -it -p 8080:8080 -p 8081:8081 --name mar -v $(pwd)/model-store:/home/model-server/model-store -v $(pwd)/examples:/home/model-server/examples pytorch/torchserve:latest
+```
+
+1.a. If starting container with Intel® Extension for PyTorch*, add the following lines in `config.properties` to enable IPEX and launcher with its default configuration.
+```
+ipex_enable=true
+cpu_launcher_enable=true
```

Comment:
is there any downside from adding `cpu_launcher_enable=true` to our config.properties by default?

Reply:
Not that I'm aware of. @ashokei, would there be any downside ?

Reply:
launcher is generic ,should not be any downside.